File: proyecto04/FrontEnd/src/app/report/interfaces/customer.ts
export interface Customer {
  id: number;
  customerName: string;
  contactLastName: string;
  contactFirstName: string;
  phone: string;
  addressLine1: string;
  addressLine2: string;
  city: string;
  state: string;
  postalCode: string;
  country: string;
  creditLimit: number;
  salesRepEmployeeNumber?: number;
}

File: proyecto04/api/src/main/java/com/example/api/src/customers/domain/Customer.java
package com.example.api.src.customers.domain;

import java.math.BigDecimal;
import lombok.Data;

@Data
public class Customer {

  private Integer id;
  private String customerName;
  private String contactLastName;
  private String contactFirstName;
  private String phone;
  private String addressLine1;
  private String addressLine2;
  private String city;
  private String state;
  private String postalCode;
  private String country;
  private BigDecimal creditLimit;
  private Employee salesRepEmployeeNumber;


}
